qutip/random_objects.py

```diff
@@ -79,21 +86,21 @@ def rand_jacobi_rotation(A, seed=None):
     angle = 2*np.random.random()*np.pi
     a = np.sqrt(0.5) * np.exp(-1j*angle)
     b = np.conj(a)
-    i = np.int(np.floor(np.random.random() * n))
+    i = np.random.randint(n)
```

Comment:
At some point we should maybe think about moving to the new (`numpy >= 1.17`) random methods, but I think we've got a few more numpy versions before they completely remove them.  I certainly don't want to do that right now.